flexmeasures/utils/app_utils.py

```diff
@@ -179,41 +179,68 @@ def parse_config_entry_by_account_roles(
+    If you load a plugin via a file path, we'll refer to the plugin with the name of your plugin folder
+    (last part of the path).
     """
-    plugin_paths = app.config.get("FLEXMEASURES_PLUGIN_PATHS", "")
-    if not isinstance(plugin_paths, list):
-        app.logger.warning(
-            f"The value of FLEXMEASURES_PLUGIN_PATHS is not a list: {plugin_paths}. Cannot install plugins ..."
+    plugins = app.config.get("FLEXMEASURES_PLUGINS", [])
+    if not plugins:
+        # this is deprecated behaviour which we should remove in version 1.0
+        app.logger.debug(
+            "No plugins configured. Attempting deprecated setting FLEXMEASURES_PLUGIN_PATHS ..."
+        )
+        plugins = app.config.get("FLEXMEASURES_PLUGIN_PATHS", [])
+    if not isinstance(plugins, list):
+        app.logger.error(
+            f"The value of FLEXMEASURES_PLUGINS is not a list: {plugins}. Cannot install plugins ..."
         )
         return
     app.config["LOADED_PLUGINS"] = {}
-    for plugin_path in plugin_paths:
-        plugin_name = plugin_path.split("/")[-1]
-        if not os.path.exists(os.path.join(plugin_path, "__init__.py")):
-            app.logger.warning(
-                f"Plugin {plugin_name} does not contain an '__init__.py' file. Cannot load plugin {plugin_name}."
+    for plugin in plugins:
+        plugin_name = plugin.split("/")[-1]
+        app.logger.info(f"Importing plugin {plugin_name} ...")
+        module = None
+        if not os.path.exists(plugin):  # assume plugin is a package
+            app.logger.debug(
+                f"Attempting to import {plugin} as an installed package ..."
             )
-            continue
-        app.logger.debug(f"Importing plugin {plugin_name} ...")
-        spec = importlib.util.spec_from_file_location(
-            plugin_name, os.path.join(plugin_path, "__init__.py")
-        )
-        if spec is None:
-            app.logger.warning(
-                f"Could not load specs for plugin {plugin_name} at {plugin_path}."
+            try:
+                module = importlib.import_module(plugin)
+            except ModuleNotFoundError:
+                app.logger.warning(
+                    f"Attempted to import module {plugin}, but it is not installed nor a valid path."
+                )
+        else:  # assume plugin is a file path
+            if not os.path.exists(os.path.join(plugin, "__init__.py")):
+                app.logger.warning(
+                    f"Plugin {plugin_name} does not contain an '__init__.py' file. Cannot load plugin {plugin_name}."
+                )
+                continue
+            spec = importlib.util.spec_from_file_location(
+                plugin_name, os.path.join(plugin, "__init__.py")
             )
+            if spec is None:
+                app.logger.warning(
+                    f"Could not load specs for plugin {plugin_name} at {plugin}."
+                )
+                continue
```

Comment:
Following up on our discussion of lines 222-225 in https://github.com/SeitaBV/flexmeasures/issues/179, I feel both those lines and these should log an error instead of a warning **(and continue),** similar to the case in line 240-241. In all three cases, the app was asked to load a plugin but it couldn't.

Reply:
I agree, that when we continue, we see an error in loading something.